pkgs/development/python-modules/fonttools/default.nix

```
@@ -24,15 +24,15 @@
 
 buildPythonPackage rec {
   pname = "fonttools";
-  version = "4.34.4";
+  version = "4.38.0";
```

```suggestion
  version = "4.38.0";
  format = "setuptools";
```

Add explicit format.